src/App.js

```diff
@@ -17,8 +18,38 @@ const tabs = [
   },
 ];
 
-const App = () => (
-  <h1>React tabs</h1>
-);
+const preparedTabs = tabs.map((tab, index) => ({
+  ...tab,
+  id: index,
+}));
+
+class App extends React.Component {
+  state = {
+    tabsOnPage: preparedTabs,
+    content: '',
+  }
+
+  handlerOnClock = (tabId) => {
```

Comment:
mb pass content?

Reply:
why is content better than id?

Reply:
because you save content in state, not id ;)
you can save it directly to state, without using find method to find it